src/client/app/ui.cljs

```diff
@@ -74,7 +74,7 @@
   (render [this]
     (let [{:keys [db/id phone/type phone/number]} (om/props this)]
       (l/row {:onClick #(om/transact! this `[(edit-phone {:id ~id})
-                                             :ui/react-key])}
+                                             :main-ui-router])}
```

Comment:
This seems like a bug in Om to me. If you use a plain prop query keyword from root, it doesn't seem to trigger a proper refresh. If you use a kw from a join, it does. I'll have to trace this down with the Om team. I've seen this before, but hadn't narrowed it down, I'd just worked around it.